### ISSUE TITLE
Proxy with Orbot returns "Method Not Allowed"

### ISSUE THREAD

**Issue by jpds** [etesync/android#124]
When I set a HTTP proxy and use Orbot's port, I get a "Method Not Allowed" attempting to add an account.

Would it be possible to potentially add support for SOCKS5 instead?

**Comment by tasn** [comment]
Yeah, it should be possible. We are currently head down focusing on EteSync 2.0, but we should find time for it next month!
